packages/firebase_dynamic_links/firebase_dynamic_links/ios/Classes/FLTFirebaseDynamicLinksPlugin.m

```diff
@@ -123,22 +123,41 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
   FLTFirebaseMethodCallErrorBlock errorBlock = ^(
       NSString *_Nullable code, NSString *_Nullable message, NSDictionary *_Nullable details,
       NSError *_Nullable error) {
+    NSMutableDictionary *temp;
     if (code == nil) {
       NSDictionary *errorDetails = getDictionaryFromNSError(error);
       code = errorDetails[kCode];
       message = errorDetails[kMessage];
-      details = errorDetails;
+
+      if (errorDetails[@"additionalData"] != nil) {
+        temp = [errorDetails[@"additionalData"] mutableCopy];
+      } else {
+        temp = [errorDetails mutableCopy];
+      }
+
+      // "NSErrorFailingURLStringKey" doesn't work
```

Comment:
Maybe we can say it doesn't work because of a Codec issue?
Might be unclear for the next dev

Reply:
Updated